### PR TITLE
Allow dynamic dark/light mode change for tooltips

### PR DESCRIPTION
**Pull request by ozone10**
Allow dynamic dark/light mode change for tooltips

fix https://github.com/notepad-plus-plus/notepad-plus-plus/issues/10059

### CODE REVIEW

File: PowerEditor/src/WinControls/FunctionList/functionListPanel.cpp [hunk 1]
```diff
@@ -871,7 +874,7 @@ INT_PTR CALLBACK FunctionListPanel::run_dlgProc(UINT message, WPARAM wParam, LPA
 			}
 
 			switch (LOWORD(wParam))
-            {
+			{
```
Comment (ozone10) [inline PowerEditor/src/WinControls/FunctionList/functionListPanel.cpp:877]: whitespace consistency

File: PowerEditor/src/WinControls/FileBrowser/fileBrowser.cpp [hunk 2]
```diff
@@ -23,7 +23,6 @@
 #include "RunDlg.h"
 #include "ReadDirectoryChanges.h"
 #include "menuCmdID.h"
-#include "Parameters.h"
```
Comment (ozone10) [inline PowerEditor/src/WinControls/FileBrowser/fileBrowser.cpp:26]: duplicate Parameters.h

File: PowerEditor/src/WinControls/TabBar/TabBar.cpp [hunk 3]
```diff
@@ -19,7 +19,6 @@
 #include <stdexcept>
 #include "TabBar.h"
 #include "Parameters.h"
-#include "NppDarkMode.h"
```
Comment (ozone10) [inline PowerEditor/src/WinControls/TabBar/TabBar.cpp:22]: already in "Parameters.h"

File: PowerEditor/src/WinControls/FileBrowser/fileBrowser.cpp (hunk 2 above)
Comment (donho) [reply PowerEditor/src/WinControls/FileBrowser/fileBrowser.cpp:26]: 👍

**Pull request closed without merging**